services/web-app/components/gif-player/gif-player.module.scss

```diff
@@ -0,0 +1,54 @@
+@use '@carbon/react/scss/theme' as theme;
+
+.container {
+  position: relative;
+}
+
+.controls {
+  position: absolute;
+  bottom: 1rem;
+  left: 1rem;
```

Comment:
Do we want spacing tokens for bottom/left?

Reply:
Done! changed for corresponding spacing tokens